configure.ac

```diff
@@ -146,6 +146,7 @@ PKG_CHECK_UPIPE(X265, x265, [x265.h])
 PKG_CHECK_UPIPE(ECORE, ecore, [Ecore.h])
 PKG_CHECK_UPIPE(ZVBI, zvbi-0.2, [libzvbi.h])
 PKG_CHECK_UPIPE(FREETYPE, freetype2, [ft2build.h])
+PKG_CHECK_UPIPE(DVBV5, libdvbv5, [libdvbv5/dvb-dev.h])
```

Comment:
The module depends on git version of libdvbv5, hopefully there's a 1.8.2 soon and we can add a version check